CODEOWNERS

```diff
@@ -19,19 +19,19 @@
 /snuba/snuba_migrations/events-analytics-platform @getsentry/events-analytics-platform @getsentry/owners-snuba
 
 # rust_snuba
-/rust_snuba @getsentry/owners-snuba @getsentry/processing
```

Comment:
I could not find `@getsentry/processing` team but `@getsentry/product-owners-processing` [team](https://github.com/orgs/getsentry/teams/product-owners-processing) exists. Should I keep `product-owners-processing` here?

Reply:
I think dropping this line is fine (since fallback is owners-snuba). Processing was helping us with Rust when we had less experience with it, but that team has been folded into ingest. They don't need special approval access anymore.